shark/src/test/java/shark/TestUtil.kt

```diff
@@ -15,7 +15,10 @@ fun <T : HeapAnalysis> File.checkForLeaks(
   referenceMatchers: List<ReferenceMatcher> = defaultReferenceMatchers,
   metadataExtractor: MetadataExtractor = MetadataExtractor.NO_OP,
   proguardMapping: ProguardMapping? = null,
-  leakFilters: List<LeakingObjectFilter> = ObjectInspectors.jdkLeakingObjectFilters
+  leakFilters: List<LeakingObjectFilter> = ObjectInspectors.jdkLeakingObjectFilters,
+  obfuscationChecker: ObfuscationChecker = object: ObfuscationChecker {
+    override fun isCodeObfuscated(): Boolean = false // assume no obfuscation in tests
```

Comment:
avoid comments at end of line they tend to wrap to next line when refactoring / formatting.